src/components/HotspotPairingList.tsx

```diff
@@ -19,6 +19,9 @@ const imagesByHotspotName: Record<string, any> = {
   'Nebra Outdoor Hotspot': (
     <Image source={require('../assets/images/nebra-outdoor-hotspot.png')} />
   ),
+  'SyncroB.it Hotspot': (
+    <Image source={require('../assets/images/syncrobit-hotspot.png')} />
```

Comment:
This is using `syncrobit-hotspot.png` here but the image names are `syncrobit_hotspot` can you update the images to use `syncrobit-hotspot`, thats the preferred pattern